### PR TITLE
Mobile UI admin page fixes

### PR DESCRIPTION
**Pull request by revengemiroz**

### CODE REVIEW

File: client/src/pages/UserTable/JsonData.js [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
 function JsonData(props) {
-  const { data = {}, loading, error } = useQuery(GET_ALL_USER_QUERY);
-
-  return {
-    loading,
-    data,
-    error,
-    allUsers: data.user || {},
-  };
+  // const { data = {}, loading, error } = useQuery(GET_ALL_USER_QUERY);
```
Comment (ivanms1) [inline client/src/pages/UserTable/JsonData.js]: remove please

File: client/src/pages/UserTable/Pagination.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      if (fetchId === fetchIdRef.current) {
+        const startRow = pageSize * pageIndex
+        const endRow = startRow + pageSize
+        setData(user.slice(startRow, endRow))
+
+        setCount(Number(pageSize*pageIndex))
+        setPageCount(Math.ceil(user.length / pageSize))
+        setLoading(false)
+        
+        console.log(count);
```
Comment (ivanms1) [inline client/src/pages/UserTable/Pagination.js]: remove

**Comment by ivanms1** [comment]
This will have to be edited once the pagination is implemented in the backend, for now it's ok.

**Comment by ivanms1** [comment]
Thi has changes from the other pr, probably forgot to switch to master before starting this pr.
@revengemiroz

File: client/src/pages/UserTable/Pagination.js (hunk 2 above)
Comment (revengemiroz) [reply client/src/pages/UserTable/Pagination.js]: done

File: client/src/pages/UserTable/JsonData.js (hunk 1 above)
Comment (revengemiroz) [reply client/src/pages/UserTable/JsonData.js]: done

**Review by ivanms1 — APPROVED** [review]

**Review by ivanms1 — APPROVED** [review]

**Pull request merged**